src/nft/components/explore/Explore.css.ts

```diff
@@ -204,7 +205,106 @@ export const valuePropContent = style([
         width: '58%',
         lineHeight: '36px',
       },
-      [`(min-width: ${breakpoints.lg}px)`]: { width: '50%' },
+      [`(min-width: ${breakpoints.lg - 1}px)`]: { width: '50%' },
+    },
+  },
+])
+
+/* Base Table Styles  */
+
+export const table = style([
+  {
+    borderCollapse: 'collapse',
+    borderStyle: 'hidden',
+    boxShadow: '0 0 0 1px rgba(153, 161, 189, 0.24)',
+    borderSpacing: '0px 40px',
+  },
+  sprinkles({
+    width: 'full',
+    borderRadius: '12',
+  }),
+])
+
+export const thead = style({
+  borderBottom: '1px solid rgba(153, 161, 189, 0.24)',
+  marginRight: '12px',
```

Comment:
these could all be sprinkles and this color is `backgroundOutline` from themes (or just `outline` if there is no change from dark mode to light mode)